src/client/components/equalizer/equalizer.js

```diff
@@ -182,31 +184,54 @@ class Equalizer extends Component {
       uploadSoundInfoFromFile,
       setCanvasToState,
     } = this;
+    const { audioData } = this.props;
```

Comment:
You can destruct everything from audio data right here, no need to create another variable, if you don’t use audio data itself

Reply:
done)